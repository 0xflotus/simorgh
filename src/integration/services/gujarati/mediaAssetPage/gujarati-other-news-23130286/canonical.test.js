/**
 * WARNING: DO NOT EDIT. This file is automatically generated by src/integration/utils/runTests/writeTestFiles/writeTestFile.js
 * @service gujarati
 * @pathname /gujarati/other-news-23130286
 */

import runCanonicalTests from '../../../../pages/mediaAssetPage/canonicalTests';

describe('Canonical gujarati mediaAssetPage', runCanonicalTests);
