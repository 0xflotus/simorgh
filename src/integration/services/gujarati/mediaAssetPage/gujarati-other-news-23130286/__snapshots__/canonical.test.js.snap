// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Canonical gujarati mediaAssetPage A11y I can see the skip to content link 1`] = `"કન્ટેન્ટ પર જાવ"`;

exports[`Canonical gujarati mediaAssetPage Analytics ATI 1`] = `"<img height=\\"1px\\" width=\\"1px\\" alt=\\"\\" style=\\"position:absolute\\" src=\\"https://logws1363.ati-host.net?s=598343&amp;s2=50&amp;p=also_in_the_news::gujarati.also_in_the_news.media_asset.23130286.page&amp;x1=[urn:bbc:cps:77bd783c-5f55-1a4f-8704-baaaee04a4e4]&amp;x2=[responsive]&amp;x3=[news-gujarati]&amp;x4=[gu]&amp;x7=[article-media-asset]&amp;x8=[simorgh-nojs]&amp;x9=[આગળની+સ્લાઇડમાં+જુઓ,+સબંધિત+વધુ+તસવીરો...+-+BBC+News+ગુજરાતી]&amp;x11=[2017-06-06T10:31:38.000Z]&amp;x12=[2017-06-06T10:31:38.000Z]&amp;x13=[નરેન્દ્ર+મોદી]&amp;x14=[103fc7e4-3a8d-491c-9a75-3c37c299d48f]&amp;x17=[News]\\"/>"`;

exports[`Canonical gujarati mediaAssetPage Footer I can click on the BBC branding and it would take me to the homepage 1`] = `"/gujarati"`;

exports[`Canonical gujarati mediaAssetPage Footer I can see the footer copyright and external linking text 1`] = `"© 2020 BBC. બહારની વેબસાઇટ્સની સામગ્રી માટે બીબીસી જવાબદાર નથી. બહારની વેબસાઇટ્સની લિંક આપવા અંગેની અમારી નીતિ વિશે વાંચો"`;

exports[`Canonical gujarati mediaAssetPage Header I can see the branding 1`] = `"BBC News, ગુજરાતી"`;

exports[`Canonical gujarati mediaAssetPage I can see the headline 1`] = `"આગળની સ્લાઇડમાં જુઓ, સબંધિત વધુ તસવીરો..."`;

exports[`Canonical gujarati mediaAssetPage I can see the timestamp 1`] = `"6 જૂન 2017"`;

exports[`Canonical gujarati mediaAssetPage Media player embed A11y I can read the media player title 1`] = `"મીડિયા પ્લેયર"`;

exports[`Canonical gujarati mediaAssetPage Media player embed I can see the media player embed 1`] = `"https://polling.test.bbc.co.uk/ws/av-embeds/cps/gujarati/other-news-23130286/p01blkgq/gu"`;

exports[`Canonical gujarati mediaAssetPage SEO Canonical link 1`] = `"http://localhost:7080/gujarati/other-news-23130286"`;

exports[`Canonical gujarati mediaAssetPage SEO Dir attribute 1`] = `"ltr"`;

exports[`Canonical gujarati mediaAssetPage SEO FB admins 1`] = `"100004154058350"`;

exports[`Canonical gujarati mediaAssetPage SEO FB app ID 1`] = `"1609039196070050"`;

exports[`Canonical gujarati mediaAssetPage SEO Heading level 1 1`] = `"આગળની સ્લાઇડમાં જુઓ, સબંધિત વધુ તસવીરો..."`;

exports[`Canonical gujarati mediaAssetPage SEO Lang attribute 1`] = `"gu"`;

exports[`Canonical gujarati mediaAssetPage SEO Linked data 1`] = `
Object {
  "@context": "http://schema.org",
  "@type": "Article",
  "about": Array [
    Object {
      "@type": "Person",
      "name": "નરેન્દ્ર મોદી",
    },
  ],
  "author": Object {
    "@type": "NewsMediaOrganization",
    "logo": Object {
      "@type": "ImageObject",
      "height": 576,
      "url": "https://news.files.bbci.co.uk/ws/img/logos/og/gujarati.png",
      "width": 1024,
    },
    "name": "BBC News ગુજરાતી",
    "noBylinesPolicy": "https://www.bbc.com/gujarati/institutional-50409883#authorexpertise",
  },
  "dateModified": "2017-06-06T10:31:38.000Z",
  "datePublished": "2017-06-06T10:31:38.000Z",
  "headline": "આગળની સ્લાઇડમાં જુઓ, સબંધિત વધુ તસવીરો...",
  "image": Object {
    "@type": "ImageObject",
    "height": 576,
    "url": "https://news.files.bbci.co.uk/ws/img/logos/og/gujarati.png",
    "width": 1024,
  },
  "mainEntityOfPage": Object {
    "@id": "http://localhost:7080/gujarati/other-news-23130286",
    "@type": "WebPage",
    "name": "આગળની સ્લાઇડમાં જુઓ, સબંધિત વધુ તસવીરો...",
  },
  "publisher": Object {
    "@type": "NewsMediaOrganization",
    "logo": Object {
      "@type": "ImageObject",
      "height": 576,
      "url": "https://news.files.bbci.co.uk/ws/img/logos/og/gujarati.png",
      "width": 1024,
    },
    "name": "BBC News ગુજરાતી",
    "publishingPrinciples": "https://www.bbc.com/gujarati/institutional-50409883",
  },
  "thumbnailUrl": "https://news.files.bbci.co.uk/ws/img/logos/og/gujarati.png",
  "url": "http://localhost:7080/gujarati/other-news-23130286",
}
`;

exports[`Canonical gujarati mediaAssetPage SEO OG description 1`] = `"ભારતીય કેપ્ટન વિરાટ કોહલીએ પાકિસ્તાન વિરૂદ્ધ જીત બાદ કહ્યું યુવરાજની પાકિસ્તાન વિરૂદ્ધ વિસ્ફોટક બેટિંગ દરમિયાન હું પોતાને 'ક્લબ બેટ્સમેન' ગણી રહ્યો હતો."`;

exports[`Canonical gujarati mediaAssetPage SEO OG image 1`] = `"http://ichef.test.bbci.co.uk/news/1024/branded_gujarati/C281/test/_63539794_p01blmqx.jpg"`;

exports[`Canonical gujarati mediaAssetPage SEO OG image alt 1`] = `"શિખર-રોહિતની કરી પ્રશંસા"`;

exports[`Canonical gujarati mediaAssetPage SEO OG locale 1`] = `"gu-IN"`;

exports[`Canonical gujarati mediaAssetPage SEO OG site name 1`] = `"BBC News ગુજરાતી"`;

exports[`Canonical gujarati mediaAssetPage SEO OG title 1`] = `"આગળની સ્લાઇડમાં જુઓ, સબંધિત વધુ તસવીરો... - BBC News ગુજરાતી"`;

exports[`Canonical gujarati mediaAssetPage SEO OG type 1`] = `"article"`;

exports[`Canonical gujarati mediaAssetPage SEO OG url 1`] = `"http://localhost:7080/gujarati/other-news-23130286"`;

exports[`Canonical gujarati mediaAssetPage SEO Page title 1`] = `"આગળની સ્લાઇડમાં જુઓ, સબંધિત વધુ તસવીરો... - BBC News ગુજરાતી"`;

exports[`Canonical gujarati mediaAssetPage SEO Robots meta tag 1`] = `"noodp,noydir"`;

exports[`Canonical gujarati mediaAssetPage SEO Twitter card 1`] = `"summary_large_image"`;

exports[`Canonical gujarati mediaAssetPage SEO Twitter creator 1`] = `"@bbcnewsgujarati"`;

exports[`Canonical gujarati mediaAssetPage SEO Twitter description 1`] = `"ભારતીય કેપ્ટન વિરાટ કોહલીએ પાકિસ્તાન વિરૂદ્ધ જીત બાદ કહ્યું યુવરાજની પાકિસ્તાન વિરૂદ્ધ વિસ્ફોટક બેટિંગ દરમિયાન હું પોતાને 'ક્લબ બેટ્સમેન' ગણી રહ્યો હતો."`;

exports[`Canonical gujarati mediaAssetPage SEO Twitter image alt 1`] = `"શિખર-રોહિતની કરી પ્રશંસા"`;

exports[`Canonical gujarati mediaAssetPage SEO Twitter image src 1`] = `"http://ichef.test.bbci.co.uk/news/1024/branded_gujarati/C281/test/_63539794_p01blmqx.jpg"`;

exports[`Canonical gujarati mediaAssetPage SEO Twitter site 1`] = `"@bbcnewsgujarati"`;

exports[`Canonical gujarati mediaAssetPage SEO Twitter title 1`] = `"આગળની સ્લાઇડમાં જુઓ, સબંધિત વધુ તસવીરો... - BBC News ગુજરાતી"`;
